mlflow/ml-package-versions.yml

```diff
@@ -143,7 +143,9 @@ lightgbm:
   package_info:
     pip_release: "lightgbm"
     install_dev: |
-      pip install git+https://github.com/microsoft/LightGBM.git#subdirectory=python-package
+      git clone --recursive https://github.com/microsoft/LightGBM --depth=1
```

Comment:
```suggestion
      git clone --recursive https://github.com/microsoft/LightGBM --depth=1 --branch master
```

Reply:
Oh , sorry, I merged too fast, I will file a follow-up PR for this.

Reply:
https://github.com/mlflow/mlflow/pull/8402